bumble/rfcomm.py

```diff
@@ -23,6 +23,7 @@
 from . import core
 from .colors import color
 from .core import BT_BR_EDR_TRANSPORT, InvalidStateError, ProtocolError
+from typing import Optional, Tuple, Callable, Dict
```

Comment:
see comment about import grouping on the other commit.

Reply:
Done